pajbot/modules/emote_limit.py

```diff
@@ -93,6 +102,11 @@ def on_message(self, source, message, emote_instances, msg_id, **rest):
 
         if len(emote_instances) > self.settings["max_emotes"]:
             self.delete_or_timeout(source, msg_id, self.settings["timeout_reason"])
+            if self.settings["moderation_action"] == "Timeout":
```

Comment:
there should definitely be a checkbox toggle for enabling or disabling the whisper